app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
```

Comment:
A comment about empty or unused ApplicationCable/ApplicationJob/etc files. I don't mind them, but if your application won't use ApplicationCable, for example, removing these files would be cleaner. You can always add them back later and it makes it easier for the person reading the code to know if you use a certain technology or not.

Reply:
Not a correction, by the way. You can keep them now because you don't know whether you're going to use it during the training or not, I'm talking more about actual projects.